src/elm_doc/elm_project.py

```diff
@@ -189,8 +187,12 @@ def iter_direct_dependencies(self) -> Iterator[ElmPackage]:
             self.direct_test_dependencies.items(),
         )
         for name, version in deps:
-            # e.g. ~/.elm/0.19.0/package/elm/core/1.0.0
-            yield from_path(elm_platform.ELM_HOME / self.elm_version / self.PACKAGES_DIRECTORY / name / version)
+            # Elm 0.19.0 uses "package" in the path, 0.19.1 uses "packages".
+            # Here we use a glob to be agnostic and somewhat defensive against
+            # future change. e.g. ~/.elm/0.19.0/package*/elm/core/1.0.0
+            elm_version_dir = elm_platform.ELM_HOME / self.elm_version
+            for elm_package_dir in elm_version_dir.glob("package*"):
```

Comment:
I believe the glob will still work if you use `package?` instead of `package*`

Reply:
Thanks for the tip. However.. at first, it seemed like `?` would work but then I realized we want something like `.?` in regex (any single character or none), while `?` in glob works like `.` in regex (any single character). That is, `package?` doesn't match `package`.